package/MDAnalysis/coordinates/chain.py

```diff
@@ -79,81 +79,105 @@ class ChainReader(base.ProtoReader):
     """
     format = 'CHAIN'
 
-    def __init__(self, filenames, **kwargs):
-        """Set up the chain reader.
+    def __init__(self, filenames, skip=1, dt=None, continuous=False, **kwargs):
+        """Set up the chain reader. This can also be used similar to `gmx trjcat` to
+        treat several parts of a long simulation as a single trajectory.
 
         Parameters
         ----------
         filenames : str or list or sequence
-           file name or list of file names; the reader will open all file names
-           and provide frames in the order of trajectories from the list. Each
-           trajectory must contain the same number of atoms in the same order
-           (i.e. they all must belong to the same topology). The trajectory
-           format is deduced from the extension of each file name.
-
-           Extension: `filenames` are either a single file name or list of file
-           names in either plain file names format or ``(filename, format)``
-           tuple combination. This allows explicit setting of the format for
-           each individual trajectory file.
+            file name or list of file names; the reader will open all file names
+            and provide frames in the order of trajectories from the list. Each
+            trajectory must contain the same number of atoms in the same order
+            (i.e. they all must belong to the same topology). The trajectory
+            format is deduced from the extension of each file name.
+
+            Extension: `filenames` are either a single file name or list of file
+            names in either plain file names format or ``(filename, format)``
+            tuple combination. This allows explicit setting of the format for
+            each individual trajectory file.
         skip : int (optional)
-           skip step (also passed on to the individual trajectory readers);
-           must be same for all trajectories
+            skip step (also passed on to the individual trajectory readers);
+            must be same for all trajectories
         dt : float (optional)
-          Passed to individual trajectory readers to enforce a common time
-          difference between frames, in MDAnalysis time units. If not set, each
-          reader's `dt` will be used (either inferred from the trajectory
-          files, or set to the reader's default) when reporting frame times;
-          note that this might lead an inconsistent time difference between
-          frames.
+            Passed to individual trajectory readers to enforce a common time
+            difference between frames, in MDAnalysis time units. If not set, each
+            reader's `dt` will be used (either inferred from the trajectory
+            files, or set to the reader's default) when reporting frame times;
+            note that this might lead an inconsistent time difference between
+            frames.
+        continuous : bool (optional)
```

Comment:
I find this bit of doc unclear. The chain reader is always used to "treat all trajectories as one single long trajectory". The point to emphasize here is that the time is made continuous as much as possible by ignoring redundant frame, only keeping the one from the latest chunk.